umclient/umclient.go

```diff
@@ -292,6 +294,13 @@ func (client *Client) processMessages() (err error) {
 	for {
 		message, err := client.stream.Recv()
 		if err != nil {
+			if code, ok := status.FromError(err); ok {
+				if code.Code() == codes.Canceled {
+					log.Info("UM client connection closed")
```

Comment:
Do we need info log in this case?

Reply:
just to know than cm was disconnected from UM.